src/config/dynamic-theme-fixes.config

```diff
@@ -822,6 +822,7 @@ allegro.pl
 INVERT
 i[title="Smart!"]
 [alt*="smart" i]
+[alt*="Smart" img]
```

Comment:
This is not correct CSS syntax, what is this supposed to fix?

Reply:
`[alt*="smart" i]`

![20221003-1664821093](https://user-images.githubusercontent.com/9846948/193649779-e00eec8e-47a8-4d3b-97a4-97299b82a7a1.png)

Don't touch this if works fine 💯

Reply:
Also ` [alt*="Smart" img] ` break all and in result logo is not inverted.  @Gusted

Reply:
uh i mean i am not that familliar with darkreader to be honest, but that fixed this banner for me

![image](https://user-images.githubusercontent.com/34240873/193651579-194d4a44-db03-4321-b562-42f61b2b0db5.png)

Reply:
@ejbtrd not all carusell images are damaged
![20221003-1664821847](https://user-images.githubusercontent.com/9846948/193652066-4ef9e762-7368-4168-97f1-6da45e511201.png)
I purged all this stuffs via Alleblock Adguard rules. This is ad, nothing more. 
Exclude can be done via select valid selector for not invert.

Reply:
@ejbtrd @Gusted Please remove your suggestion and try
```
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
```

Reply:
@ejbtrd second 
```
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
```

Reply:
CC @Gusted ping

Reply:
```suggestion
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
```

Reply:
@ejbtrd where is banner with Allegro Lokanie? I cannot find it. 🤔

Reply:
@Gusted in a free time I will prepare new allegro fixes - some fonts are to dimmed.

Reply:
> @ejbtrd @Gusted Please remove your suggestion and try
> 
> ```
> ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
> ```

This selector fixes Smart Week banner on main page, however the whole issue started from https://allegro.pl/smart.

Right now i have also noticed that many more images are incorrectly inverted:

![image](https://user-images.githubusercontent.com/34240873/193656223-b09f0e7c-0347-4702-9ba5-09c0ed807edb.png)

Reply:
@Gusted @ejbtrd please wait 💯

Reply:
ping

Reply:
CC @Gusted ping

Reply:
So:
```suggestion
ol[data-role="slidesList"] > li[data-analytics-view-value="Smart! Week już od 10 października"]
ul[data-role="image-tiles"] > li[data-analytics-view-opbox-custom-destination-url="https://allegro.pl/kampania/smartweek"]
a[data-analytics-click-value="Smart lokalnie"]
```

Would be correct?

Reply:
@Gusted for this "issues" for a while yes. 
Allegro changes many times banners codes, sadly not universal catch work here.